erpnext/quality_management/doctype/quality_meeting/test_quality_meeting.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe and Contributors
+# See license.txt
+from __future__ import unicode_literals
+
+import frappe
+import unittest
+
+class TestQualityMeeting(unittest.TestCase):
+	def test_quality_meeting(self):
+		test_create_meeting = create_meeting()
+		test_get_meeting = get_meeting()
+		self.assertEquals(test_create_meeting.name, test_get_meeting.name)
+
+def create_meeting():
+	meeting = frappe.get_doc({
+		"doctype": "Quality Meeting",
+		"scope": "Company",
+		"status": "Close",
+		"date": ""+ frappe.utils.nowdate() +""
```

Comment:
`frappe.as_unicode(frappe.utils.nowdate())`

Reply:
changed it